sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -522,15 +579,16 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [metrics_advisor_client_class]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs
 
 [metricsadv-sample1]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample01_DataFeedCrudOperations.cs
-[metricsadv-sample2]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample02_DataFeedIngestionOperations.cs
```

Comment:
Wanted to put the credential entity sample right after the data feed one, so moved everything down.